multiqc/multiqc.py

```diff
@@ -233,7 +233,17 @@
     is_flag=True,
     help="Use only interactive plots [i](in-browser Javascript)[/]",
 )
-@click.option("--lint", "lint", is_flag=True, help="Use strict linting (validation) to help code development")
+@click.option(
+    "--strict",
+    "--lint",
```

Comment:
I think have this as a separate `@click.option`, and merge in the Python code of the function. Then the CLI option can be marked as `hidden` with Click and not show up in the CLI help. Can also throw a warning log message in the Python of the main def saying that it's deprecated.

Reply:
Yes, good idea!